gitlab-docker/README.md

```diff
@@ -164,6 +170,33 @@ Navigate to "http://localhost:8083/" and log in with the name and password in
 the `my-vars.sh` file.
 
 Use the `+` button at the bottom right to create a new bucket called `spack-public`.
+To use this service as a binary mirror in your build pipelines, the mirror url
+you should set will depend on the bucket name you chose.  If using `spack-public`
+as the bucket name, then configure the mirror in your release environment as
+follows:
+
+```
+  mirrors: { "mirror": "s3://spack-public/mirror" }
+```
+
+Use of S3 for binary mirror requires access credentials, which should be supplied
+via environment variables available to your build jobs.  Navigate to
+`localhost:8080/root/spack-build/-/settings/ci_cd`, and under "Variables", set
+up the following:
+
+```
+AWS_ACCESS_KEY_ID=minio
+AWS_SECRET_ACCESS_KEY=minio123
+```
+
+Substitute whatever values you provided for `MINIO_ACCESS_KEY` and `MINIO_SECRET_KEY`
+in the `docker-compose.yml` file.  Additionally, since this is not an AWS S3 service,
+the `boto3` module requires that you specify an endpoint url in your environment.  In
```

Comment:
Minor correction: the fact that the endpoint url needs to be specified in an environment variable is due to our approach to handling S3 urls; not boto, per se.  Also, it may or may not be relevant to mention that the `AWS_ENDPOINT_URL` variable name is a fictional name that I came up with and will likely change to `S3_ENDPOINT_URL`.